daemon/cmd/server/main.go

```diff
@@ -171,14 +171,20 @@ func main() {
 	daemonBackgroundJob := daemon.Job{Cleaner: cleaner, Reminder: daemon.Reminder{}, Investment: daemon.Investment{}}
 	log.Infof("The next daemon job will start at %v", start.Format(time.RFC3339))
 	log.Infof("And Now it's %v", time.Now().Format(time.RFC3339))
+	
+	var interval time.Duration
+	interval = time.Hour * 24 * time.Duration(daemonRpc.serviceConfig.IntervalInDays)
```

Comment:
year, month, day := time.Now().AddDate(0, 0, daemonRpc.serviceConfig.IntervalInDays).In(PST).Date()  start := time.Now()
 start := time.Date(year, month, day, 0, 0, 0, 0, PST)

Reply:
Please fix start

Reply:
Also format code